kafkajobs/opts.go

```diff
@@ -119,3 +120,9 @@ type ConsumerOpts struct {
 	ConsumePartitions   map[string]map[int32]*Offset `mapstructure:"consume_partitions" json:"consume_partitions"`
 	ConsumerOffset      *Offset                      `mapstructure:"consumer_offset" json:"consumer_offset"`
 }
+
+type SSL struct {
+	Enabled            bool          `mapstructure:"enabled" json:"enabled"`
```

Comment:
Could you please tell, me why do you need this option instead of just checking for the `c.SSL` for `nil`.